src/main/scala/za/co/absa/abris/config/Config.scala

```diff
@@ -27,6 +27,7 @@ object AbrisConfig {
   def fromConfluentAvro: FromConfluentAvroConfigFragment = new FromConfluentAvroConfigFragment()
 
   val SCHEMA_REGISTRY_URL = "schema.registry.url"
+  val SCHEMA_REGISTRY_CLASS = "schema.registry.class"
```

Comment:
```suggestion
  val REGISTRY_CLIENT_CLASS = "spline.registryClient.class"
```
`spline` to avoid any name conflicts and its a client...